src/test/resources/feature/3-destinatario/personaFisica/deleghe/005_048_accettazioneDelega.feature

```diff
@@ -8,11 +8,25 @@ Feature: il delegato accetta la delega
   Scenario: PN-9411 - il delegato accetta la delega
     Given PF - Si effettua la login tramite token exchange come "delegato", e viene visualizzata la dashboard
     When Nella pagina Piattaforma Notifiche persona fisica click sul bottone Deleghe
-    And Si verifica sia presente una delega nella sezione Deleghe a Tuo Carico
-      | nome          | Gaio Giulio |
-      | cognome       | Cesare      |
+    When Creo in background una delega per persona fisica
+      | accessoCome | delegante        |
+      | fiscalCode  | BRGLRZ80D58H501Q |
+      | person      | true             |
+      | displayName | Lucrezia Borgia  |
+      | firstName   | Lucrezia         |
+      | lastName    | Borgia           |
+    And Si verifica che presente un indicatore numerico in corrispondenza della voce di menù Deleghe
     And Si sceglie opzione accetta
-    And Si inserisce il codice delega nel pop-up "nuova_delega"
+    And  Si clicca sul bottone indietro popup
+    And Si sceglie opzione accetta
+    And Si inserisce il codice delega nel pop-up OTP "errato"
+    And Si clicca sul bottone Accetta
```

Comment:
verify exist method that create delegation, and if it exist replace with single step

Reply:
ma questo e metodo ge hai dettto usare tu. Altri metodi non vanno bene, perche dopo la creazione di delega, prima di accettare, dobbiamo aprire il popup , chiudere e riaprire per accettare

Reply:
okok